conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+class FastNoiseLiteConan(ConanFile):
+    name = "fastnoise-lite"
+    version = "2021.11.29"
```

Comment:
Just writing this comment so we don't forget to change the versioning

Reply:
1.0.3.2111 ?